src/frequenz/sdk/timeseries/logical_meter/_formula_generators/_ev_charger_power_formula.py

```diff
@@ -0,0 +1,53 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Formula generator from component graph for Grid Power."""
+
+import logging
+
+from .....sdk import microgrid
+from ....microgrid.component import ComponentCategory, ComponentMetricId
+from .._formula_engine import FormulaEngine
+from ._formula_generator import NON_EXISTING_COMPONENT_ID, FormulaGenerator
+
+logger = logging.getLogger(__name__)
+
+
+class EVChargerPowerFormula(FormulaGenerator):
+    """Create a formula engine from the component graph for calculating grid power."""
+
+    async def generate(self) -> FormulaEngine:
+        """Generate a formula for calculating total EV power from the component graph.
+
+        Returns:
+            A formula engine that calculates total EV charger power values.
+        """
+        builder = self._get_builder("ev-power", ComponentMetricId.ACTIVE_POWER)
+        component_graph = microgrid.get().component_graph
+        ev_chargers = [
+            comp
+            for comp in component_graph.components()
+            if comp.category == ComponentCategory.EV_CHARGER
+        ]
```

Comment:
@sahas-subramanian-frequenz  should we check if there is meter before ev_chargers?
And if there is a meter we could get active_power from the meter.

Reply:
Let's do that after we have a better idea of what all the possible layouts are.

It is all resampled data that we are operating on here,  so it should be minimal difference.